speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextIT.java

```diff
@@ -395,7 +395,7 @@ public void onTranscription(SpeechRecognitionResults speechResults) {
             .getWordAlternatives();
     assertTrue(wordAlternatives != null && !wordAlternatives.isEmpty());
     assertNotNull(wordAlternatives.get(0).getAlternatives());
-    assertNotNull(asyncTranscriptionResults.getProcessingMetrics());
+    // assertNotNull(asyncTranscriptionResults.getProcessingMetrics());
```

Comment:
commented code

Reply:
fixed in [f6d5c6c8042b4fdda54d69d64566dd14afef05b8](https://github.com/watson-developer-cloud/java-sdk/pull/1128/commits/f6d5c6c8042b4fdda54d69d64566dd14afef05b8)